### PR TITLE
docs(faq): update content for SEO optimization MTA-1762

### PR DESCRIPTION
**Pull request by nedaehm**

### CODE REVIEW

File: faq/elastic-metal.mdx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Elastic Metal servers
-  description: Let us address the most common questions you have about Elastic Metal servers.
+  description: Discover Scaleway Elastic Metal and Bare Metal servers.
```
Comment (nedaehm) [inline faq/elastic-metal.mdx:4]:
```suggestion
  description: Discover Scaleway Elastic Metal servers.
```

File: faq/elastic-metal.mdx [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 content:
   h1: Elastic Metal servers
-  paragraph: Let us address the most common questions you have about Elastic Metal servers.
+  paragraph: Discover Scaleway Elastic Metal and Bare Metal servers.
```
Comment (nedaehm) [inline faq/elastic-metal.mdx:7]:
```suggestion
  paragraph: Discover Scaleway Elastic Metal servers.
```

File: faq/instances.mdx [hunk 3]
```diff
@@ -1,18 +1,24 @@
 ---
 meta:
   title: Instances
-  description: Let us address the most common questions you have about Instances.
+  description: Discover Scaleway Instances.
```
Comment (nedaehm) [inline faq/instances.mdx:4]:
```suggestion
  description: Discover Scaleway Instances and learn the difference between image and snapshot.
```

File: faq/elastic-metal.mdx [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Elastic Metal servers
-  description: Let us address the most common questions you have about Elastic Metal servers.
+  description: Discover Scaleway Elastic Metal servers.
```
Comment (nedaehm) [inline faq/elastic-metal.mdx:4]:
```suggestion
  description: Discover Scaleway Elastic Metal servers, the first 100% cloud bare metal servers.
```

File: faq/apple-silicon-m1-as-a-service.mdx [hunk 5]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Apple silicon M1 as-a-service
-  description: Let us address the most common questions you have about Apple silicon M1 as-a-service.
+  description: Discover how to use M1 for your OS applications.
```
Comment (bene2k1) [inline faq/apple-silicon-m1-as-a-service.mdx:4]:
```suggestion
  description: Discover how to use M1 for applications development.
```

File: faq/apple-silicon-m1-as-a-service.mdx [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 content:
   h1: Apple silicon M1 as-a-service
-  paragraph: Let us address the most common questions you have about Apple silicon M1 as-a-service.
+  paragraph: Discover how to use M1 for your OS applications.
```
Comment (bene2k1) [inline faq/apple-silicon-m1-as-a-service.mdx:7]:
```suggestion
  paragraph: Discover how to use M1 for applications development.
```

File: faq/apple-silicon-m1-as-a-service.mdx [hunk 7]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Apple silicon M1 as-a-service
-  description: Let us address the most common questions you have about Apple silicon M1 as-a-service.
+  description: Discover how to use M1 for applications development.
```
Comment (nedaehm) [inline faq/apple-silicon-m1-as-a-service.mdx:4]:
```suggestion
  description: Discover how to use M1 for applications' development.
```

File: faq/apple-silicon-m1-as-a-service.mdx [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
 content:
   h1: Apple silicon M1 as-a-service
-  paragraph: Let us address the most common questions you have about Apple silicon M1 as-a-service.
+  paragraph: Discover how to use M1 for applications development.
```
Comment (nedaehm) [inline faq/apple-silicon-m1-as-a-service.mdx:7]:
```suggestion
  paragraph: Discover how to use M1 for applications' development.
```

File: faq/apple-silicon-m1-as-a-service.mdx [hunk 9]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Apple silicon M1 as-a-service
-  description: Let us address the most common questions you have about Apple silicon M1 as-a-service.
+  description: Discover how to use M1 for applications' development.
```
Comment (RoRoJ) [inline faq/apple-silicon-m1-as-a-service.mdx:4]: `Discover how to use M1 for application development` would sound better I think?

File: faq/database.mdx [hunk 10]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Database
-  description: Let us address the most common questions you have about Database.
+  description: Discover Scaleway managed Database and how to access it for PostgreSQL and MySQL.
```
Comment (RoRoJ) [inline faq/database.mdx:4]: Managed Database (and in the title below too)

File: faq/elastic-metal.mdx [hunk 11]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Elastic Metal servers
-  description: Let us address the most common questions you have about Elastic Metal servers.
+  description: Discover Scaleway Elastic Metal servers, the first 100% cloud bare metal servers.
```
Comment (RoRoJ) [inline faq/elastic-metal.mdx:4]: Is it normal that we have bare metal written here? I guess it's for SEO?

File: faq/instances.mdx [hunk 12]
```diff
@@ -1,18 +1,24 @@
 ---
 meta:
   title: Instances
-  description: Let us address the most common questions you have about Instances.
+  description: Discover Scaleway Instances and learn the difference between image and snapshot.
```
Comment (RoRoJ) [inline faq/instances.mdx:4]:
```suggestion
  description: Discover Scaleway Instances and learn the difference between images and snapshots.
```

File: faq/serverless-containers.mdx [hunk 13]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Serverless Containers
-  description: Let us address the most common questions you have about Serverless Containers.
+  description: Discover Faas with Serverless Containers.
```
Comment (RoRoJ) [inline faq/serverless-containers.mdx:4]:
```suggestion
  description: Discover FaaS with Serverless Containers.
```

File: faq/serverless-containers.mdx [hunk 14]
```diff
@@ -1,10 +1,10 @@
...
 content:
   h1: Serverless Containers
-  paragraph: Let us address the most common questions you have about Serverless Containers.
+  paragraph: Discover Faas with Serverless Containers.
```
Comment (RoRoJ) [inline faq/serverless-containers.mdx:7]:
```suggestion
  paragraph: Discover FaaS with Serverless Containers.
```

File: faq/apple-silicon-m1-as-a-service.mdx [hunk 15]
```diff
@@ -1,10 +1,10 @@
...
 content:
   h1: Apple silicon M1 as-a-service
-  paragraph: Let us address the most common questions you have about Apple silicon M1 as-a-service.
+  paragraph: Discover how to use M1 for applications' development.
```
Comment (nedaehm) [inline faq/apple-silicon-m1-as-a-service.mdx:7]:
```suggestion
  paragraph: Discover how to use M1 for application development.
```

File: faq/database.mdx (hunk 10 above)
Comment (nedaehm) [inline faq/database.mdx:4]:
```suggestion
  description: Discover Scaleway Managed Database and how to access it for PostgreSQL and MySQL.
```

File: faq/database.mdx [hunk 16]
```diff
@@ -1,10 +1,10 @@
...
 content:
   h1: Database
-  paragraph: Let us address the most common questions you have about Database.
+  paragraph: Discover Scaleway managed Database and how to access it for PostgreSQL and MySQL.
```
Comment (nedaehm) [inline faq/database.mdx:7]:
```suggestion
  paragraph: Discover Scaleway Managed Database and how to access it for PostgreSQL and MySQL.
```

File: faq/elastic-metal.mdx (hunk 11 above)
Comment (nedaehm) [reply faq/elastic-metal.mdx:4]: Yes, Vilena suggested this one

**Review by RoRoJ — APPROVED** [review]

File: faq/instances.mdx [hunk 17]
```diff
@@ -38,7 +44,7 @@ When you select the **Standby** option, your instance is stopped, but all your d
 With the Standby option, you are still charged the same amount as if the instance is running.
 </Message>
 
-## Can I reach my Instance using a DNS hostname instead of it's IP address?
+## Can I reach my Instance using a DNS hostname instead of its IP address?
 
 Yes, each instance has two DNS hostnames associated with it:
```
Comment (nedaehm) [inline faq/instances.mdx:49]:
```suggestion
Yes, each Instance has two DNS hostnames associated with it: 
```

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**